### PR TITLE
fix: double scroll bars in chart grid

### PR DESCRIPTION
**Pull request by keita-determined**
## Description
[WEB-932]

https://user-images.githubusercontent.com/109113616/227026615-5e9b55ca-7ec0-4b04-a593-1c7d0cbb1bb6.mov

## Test Plan
- Go to experiment overview page
- Enable chart flag by adding `f_chart=on` if we still use chart flag
- Narrow screen horizontally
- Ideally there are at least 3 charts
- Check if no double scroll bars even if screen width is changed

## Checklist

- [x] Changes have been manually QA'd
- [ ] User-facing API changes need the "User-facing API Change" label.
- [ ] Release notes should be added as a separate file under `docs/release-notes/`.
  See [Release Note](https://github.com/determined-ai/determined/blob/master/docs/release-notes/README.md) for details.
- [ ] Licenses should be included for new code which was copied and/or modified from any external code.

## Ticket

[WEB-932]: https://hpe-aiatscale.atlassian.net/browse/WEB-932?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: webui/react/src/pages/TrialDetails/F_TrialDetailsOverview.tsx [hunk 1]
```diff
@@ -234,7 +234,9 @@ const TrialDetailsOverview: React.FC<Props> = ({ experiment, trial }: Props) =>
           )}
         </>
       ) : null}
-      {contextHolders}
+      {contextHolders.map((contextHolder, i) => (
+        <React.Fragment key={i}>{contextHolder}</React.Fragment>
+      ))}
```
Comment (hkang1) [inline webui/react/src/pages/TrialDetails/F_TrialDetailsOverview.tsx:239]: I think this works, but doesn't each `contextHolder` themselves already have a `<>` wrapper around them? I don't have anything better to suggest, so if this works to get rid of the missing key warning 👍

**Review by hkang1 — APPROVED** [review]
The double scroll bar fix looks good.

File: webui/react/src/hooks/useModal/Model/useModalModelEdit.tsx [hunk 2]
```diff
@@ -63,7 +63,7 @@ const useModalModelEdit = ({ onClose, model, fetchModel }: Props): ModalHooks =>
             <Input />
           </Form.Item>
           <Form.Item initialValue={model.description} label="Description" name="description">
-            <Input />
+            <Input.TextArea />
```
[the remark this answers is not in the document]
Comment (keita-determined) [reply webui/react/src/hooks/useModal/Model/useModalModelEdit.tsx:66]: oh it should be in different pr to make this PR clear. i'll revert it

File: webui/react/src/components/kit/LineChart.tsx [hunk 3]
```diff
@@ -356,6 +356,7 @@ export const ChartGrid: React.FC<GroupProps> = React.memo(
                   itemData={{ chartsProps: chartsProps, columnCount, scale, xAxis }}
                   rowCount={Math.ceil(chartsProps.length / columnCount)}
                   rowHeight={480}
+                  style={{ height: '100%' }}
```
[the remark this answers is not in the document]
Comment (keita-determined) [reply webui/react/src/components/kit/LineChart.tsx:359]: no, also it doesnt accept string type to pass `100%`, so css is the easiest way to handle this

File: webui/react/src/pages/TrialDetails/F_TrialDetailsOverview.tsx (hunk 1 above)
Comment (keita-determined) [reply webui/react/src/pages/TrialDetails/F_TrialDetailsOverview.tsx:239]: the suggestion didnt work 🥲

**Pull request merged**